qprofiler/src/org/qcmg/qprofiler/fastq/FastqSummaryReport.java

```diff
@@ -292,18 +292,18 @@ void parseFiveElementHeaderWithSpaces(String [] params) {
 		// split by space
 		String [] firstElementParams = params[0].split(" ");
 		if (firstElementParams.length != 2) {
-			throw new UnsupportedOperationException("Incorrect header format encountered in parseFiveElementHeader. Expected '@ERR091788.3104 HSQ955_155:2:1101:13051:2071/2' but recieved: " + Arrays.deepToString(params));
+			throw new UnsupportedOperationException("Incorrect header format encountered in parseFiveElementHeader. Expected '@ERR091788.3104 HSQ955_155:2:1101:13051:2071/2' but received: " + Arrays.deepToString(params));
```

Comment:
Here and ll 299, 306 the Exception messages are a bit confusing because they refer not to a generic format but to a specific example as 'Expected'; is there a better wording for these?